R/vdiagram-render.R

```diff
@@ -67,7 +67,10 @@ createVdiagram <- function(data) {
 
   vplot <- testCallouts(vplot, xlimits = xlim(vplot)$side.1)
   
-  ylims <- ylim(vplot)$side.2
+  ylims <- c(
+      min(c(ylim(vplot)$side.2, vdiagramData$minStage)),
+      max(c(ylim(vplot)$side.2, vdiagramData$maxStage))
+      )
```

Comment:
The latter solution certainly make sense "on paper", but doesn't produce the expected behavior when implemented, and I don't know what at the moment.